Makefile

```diff
@@ -5,6 +5,7 @@
 SHELL=/bin/sh
 CONTAINER_COMMAND = $(shell if [ -x "$(shell command -v docker)" ];then echo "docker" ; else echo "podman";fi)
 PULL_PARAM=$(shell if [ "${CONTAINER_COMMAND}" = "podman" ];then echo "--pull-always" ; else echo "--pull";fi)
+CI ?= true
```

Comment:
what effect does it have and how is it connected to kube-api test?

Reply:
It's override the definition on assisted-service Makefile so that when the service is deployed it will have all the available images.
It's not related to kubeapi test directly but the test run on 4.8 and I wanted that test-infra will take the default value from assisted-service and not form other places